RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate REST API - saveRoomSettings to Typescript

## Proposed changes

Chore: Migrate REST API - saveRoomSettings to Typescript

## Issue(s)	


## How to test or reproduce

- Create a channel
- Enter with your E2E password
- Click in the header of the room
- If appear a toggle button Encrypted: enable and disable the button
- Or click on the card with name of the room
- inside the RoomInfoView
- Click to Edit the room
- Edit the room and save

## Screenshots

## Types of changes




## Checklist



## Further comments
